ferrilata-devilline/Services/Helpers/Extensions/DatabaseSeeder.cs

```diff
@@ -66,6 +74,16 @@ private static List<Level> CreateLevels(ApplicationContext context, List<Badge>
             return context.Levels.OrderBy(b => b.LevelId).ToList();
         }
 
+        private static List<Badge> UpdateBadgesWithLevels(ApplicationContext context, List<Badge> badges, List<Level> levels)
+        {
+            badges[0].Levels = new List<Level> { levels[0] };
+            badges[1].Levels = new List<Level> { levels[1] };
+
+            context.Badges.UpdateRange(badges);
+            context.SaveChanges();
+            return badges;
```

Comment:
For the update method, it looks like there is no need to **return** the updated list.